duneservice.py

```diff
@@ -98,25 +98,23 @@ def cancel_query_execution(self, execution_id):
 
         return response
 
-    def post_process_query_result(self, response):
-        post_processed_result = {
-            'wallet': self.wallet,
-            'total_transfers': 0,
-            'blocks': []
-        }
+    def post_process_query_result(self, response, start_block, end_block):
```

Comment:
The returned dune rows have a simplified schema. It's just "from" and "transfers". "transfers" is summed across the block range.